fv3core/stencils/dyn_core.py

```diff
@@ -364,6 +364,85 @@ def __init__(
             domain=self.grid.domain_shape_full(add=(0, 0, 1)),
         )
 
+        # Halo updaters
```

Comment:
can we collect all of those in a class?

Reply:
Pushed `HaloUpdaters `subclass in `AcousticDynamics`